_episodes/02-loop.md

```diff
@@ -141,6 +141,14 @@ for variable in collection:
 ~~~
 {: .language-python}
 
+> ## What is the ":" for?
+>
+> One thing you may have noticed is that our `for` statement ends with a `:`. You actually will see the `:` regularly in Python, and
+> it actually can serve two functions. The first is indexing, which you became familiar with in the previous episode. The second, as is
+> the case in `for` loops, is the opening of a "block". A block is one or more lines of code that are indented from their parent and
```

Comment:
`"block"` is jargon and, in this case, ambiguous. I know what you mean but those who are just starting with Python - don't.

`of a "block"` -> `of the indented block of code that follows`.

Reply:
`parent`  jargon as well.

There is simply too much cognitive load in this paragraph.